apsuite/commissioning_scripts/measure_respmat.py

```diff
@@ -19,13 +19,19 @@ def __init__(self):
         """."""
         self.nr_points_smooth = 50
         self.corr_nr_iters = 10
+        self.delta_kickx = 15  # [urad]
+        self.delta_kicky = 15  # [urad]
+        self.delta_rf = 80     # [Hz]
         self.respmat_name = ''
 
     def __str__(self):
         """."""
         dtmp = '{0:26s} = {1:9d}  {2:s}\n'.format
         stg = dtmp('nr_points_smooth', self.nr_points_smooth, '')
         stg += dtmp('corr_nr_iters', self.corr_nr_iters, '')
+        stg += dtmp('delta_kickx', self.delta_kickx, '[urad]')
```

Comment:
I think this is wrong. The kicks are floats, not integers. I think you have to create a string template for floats. I suggest using two decimal places.

Reply:
Indeed, thanks!